plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -227,3 +228,12 @@ runPlcCekTrace values = do
         UPLC.runCek PLC.defaultCekParameters UPLC.tallying UPLC.logEmitter (p ^. UPLC.progTerm)
   res <- fromRightM (throwError . SomeException) result
   pure (logOut, tally, res)
+
+-- | A group of golden tests including UPLC size and UPLC and PIR readable.
+goldenTests :: String -> CompiledCode a -> [TestTree]
+goldenTests testName code =
+  [
+  runTestNested $ goldenSize testName code,
+  runTestNested $ goldenUPlcReadable testName code,
```

Comment:
personally I don't think that including the UPLC output is usually helpful. The final PIR is generally good enough

Reply:
The golden UPLC files change with the new UPLC inliner code for the other `plutus-tx-tests` so I definitely want the UPLC golden files.

Reply:
Why these tests though? In the plutus tx tests we mostly look at the PIR, we have a few exceptions but they should be justified, though. It seems unlikely to me that we're regularly going to want to look at the UPLC here?

Reply:
There are regressions in some of these tests. I didn't think it's costly to add more golden files, so I did it to all of these tests. Next time it may be other tests that will have regression so I was hoping to save us time. Also, all in all it'll be good to know that the UPLC terms haven't changed anyway.

Reply:
It's a bit costly, in that it's more stuff to look through (and usually ignore). If there's an interesting example I'd be inclined to just add that one to the optimization tests.

Reply:
Ok sure will do.